### ISSUE TITLE
Can't override ErrorHandler in New(Options)

### ISSUE THREAD

**Issue by jaimem88** [auth0/go-jwt-middleware#51]
jwtmiddleware.Options struct has an ErrorHandler option but the type is `errorHandler` which is not exported. Can't override with a custom error Handler.

**Comment by adamo57** [comment]
I assume you could just make your own `errorHandler` and pass it to jwtmiddleware.Options.ErrorHandler

```
type errorHandler func(w http.ResponseWriter, r *http.Request, err string)

**Comment by sergiughf** [comment]
We just released the [v2.0.0-beta](https://github.com/auth0/go-jwt-middleware/releases/tag/v2.0.0-beta) 🥳 ! 

You can start testing it by running `go get github.com/auth0/go-jwt-middleware/v2@v2.0.0-beta`. 

In case of issues fetching the v2 you might want to try `go clean --modcache` first before doing `go get`.

I'm closing this issue as now this is part of v2, but feel free to reopen if needed.
